fields/types/gcsavatar/GcsAvatarField.js

```diff
@@ -8,7 +8,7 @@ const _ = {
   get,
 };
 
-const supportTypes = ['image/gif', 'image/png', 'image/jpeg', 'image/x-tiff', 'image/svg+xml', 'image/bmp'];
```

Comment:
Would we want to change the alert at Ln78 and note at Ln111 as well?

Reply:
Updated, thanks for the reminder.